variables.tf

```diff
@@ -1,9 +1,59 @@
-########################################################################################################################
+##############################################################################
 # Input Variables
-########################################################################################################################
+##############################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to use"
+  type        = string
+  sensitive   = true
+  default     = null
+}
+
+variable "region" {
+  description = "Cluster region"
+  type        = string
+  nullable    = false
+}
```

Comment:
region should not be needed at the root level module? Its only needed for provider config which is only added for examples or DAs